src/main/java/ca/team2706/frc/robot/commands/drivebase/RotateWithGyro.java

```diff
@@ -0,0 +1,83 @@
+package ca.team2706.frc.robot.commands.drivebase;
+
+import ca.team2706.frc.robot.subsystems.DriveBase;
+import edu.wpi.first.wpilibj.command.Command;
+
+import java.util.function.Supplier;
+
+public class RotateWithGyro extends Command {

```

Comment:
We made a new class called DriveBaseClosedLoop which should now be extended instead of Command